server/src/local/organisasajonerMockdata.ts

```diff
@@ -106,14 +98,6 @@ export const organisasjonerMedIaRettighet = [
     Status: "Active",
     ParentOrganizationNumber: "",
   },
-  {
-    Name: "Krever innlogging",
```

Comment:
Dene ga ikke så mye mening å ha. Hvis vi klarer å hente liste over virsomheter, så betyr det jo at vi allerede er innlogget 🤷‍♂️

Reply:
dette er vel for å mocke `401`-respons i `proxyMiddlewareMock.ts`? typ velg denne bedriften for å bli redirectet til innlogging/teste den rest-statusen i frontend